### PR TITLE
Create components for voting page

### LINKED ISSUE #25: Voting Interface Page
# Description

Create Voting Interface Page and necessary components

# Related to user story

#3 #4 #6 #7 #8

# Notes and resources

https://lottiefiles.com/8821-like-button?lang=de
https://www.joshwcomeau.com/react/boop/

# Tasks

- [x] Episode card item (manage overflow of longer titles with three dots)
- [x] Burger menu icon
- [x] Filter icon with hover animation
- [x] Input for search with suggested results
- [x] Like button and counter
- [x] Set up LottieFile animation for likes
- [x] Check and adjust responsiveness
- [x] Create page
- [x] Create stories for page and components

### PR DESCRIPTION
**Pull request by ncbecker**
## Description

- Episode Card
- Search Field
- Create Page and Story

## Related Issue

Closes #25

### CODE REVIEW

**Comment by kschmitz123** [comment]
lgtm 🚀 i like the effect when you click on "like"

File: client/src/assets/icon-heart-empty.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg id="icon-heart-empty-green" xmlns="http://www.w3.org/2000/svg" width="35" height="33.675" viewBox="0 0 35 33.675">
+  <path id="icon-heart-empty-green-2" data-name="icon-heart-empty-green" d="M17.5,33.675C8.09,26.951,3.155,20.834,1.131,15.715-4.29,2,11.2-4.565,17.5,3.533,23.8-4.565,39.29,2,33.869,15.715,31.845,20.834,26.91,26.951,17.5,33.675Z" fill="rgba(51,51,51,0.5)"/>
+</svg>
```
Comment (lmachens) [inline client/src/assets/icon-heart-empty.svg:3]: You can reduce the file size by removing unused fields like `id` or `data-name`.
These tools might be helpful:
https://www.svgminify.com/
https://imageoptim.com/

File: client/src/assets/lotties/heart-burst.json [hunk 2]
```diff
@@ -0,0 +1,211 @@
+{
```
Comment (lmachens) [inline client/src/assets/lotties/heart-burst.json:1]: If you use lottie files, please attribute the author.
https://lottiefiles.com/page/license

As long as this app is not available for end users, I would add a LICENSES file or add a comment to the top of this file.
When you release this app, consider to add a licenses section in your app.

File: client/src/components/EpisodeCard.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  return (
+    <Card>
+      <EpisodeInfos>
+        <img src={Placeholder} alt="Placeholder" />
+        <span>Sehnsucht nach New York / Fernsehabend mit Jan und Olli</span>
+      </EpisodeInfos>
+      <LikeContainer>
+        <LikeButton onClick={handleClickLike}>
+          {!isLiked && <NotLiked />}
+          {isLiked && <Liked />}
```
Comment (lmachens) [inline client/src/components/EpisodeCard.js]: If you like, you can write this in one line
```suggestion
          {isLiked ? <Liked /> : <NotLiked />}
```

File: client/src/components/EpisodeCard.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  color: var(--button-text-light);
+`;
+
+const LottieContainer = styled.div`
+  position: absolute;
+  top: -180px;
+  left: -180px;
+`;
+
+export const EpisodeCard = () => {
```
Comment (lmachens) [inline client/src/components/EpisodeCard.js]: Right now this component always renders the same episode ("Sehnsucht nach New York"). Next, it's a good idea to add properties.

**Review by lmachens — COMMENTED** [review]
gj adding lottie animations 👍.
This pull request has many unrelated changes like removing of sample components.
It's easier to review if you split it in multiple pull requests next time (it's fine now).

Your components have no properties right now, thus, the stories can not show different usages. I would love to see this soon :).

File: client/src/assets/icon-heart-empty.svg (hunk 1 above)
Comment (ncbecker) [reply client/src/assets/icon-heart-empty.svg:3]: Fixed in commit https://github.com/ncbecker/podium/pull/42/commits/98fe3499b42fecbebde58e42a53fa256cd6e26d5
Thanks 👍

File: client/src/assets/lotties/heart-burst.json (hunk 2 above)
Comment (ncbecker) [reply client/src/assets/lotties/heart-burst.json:1]: Fixed in commit https://github.com/ncbecker/podium/pull/42/commits/65f9303d68254705fab3b1d778f57e0ab920586f
Thanks 👍

File: client/src/components/EpisodeCard.js (hunk 3 above)
Comment (ncbecker) [reply client/src/components/EpisodeCard.js]: Adjusted in commit https://github.com/ncbecker/podium/pull/42/commits/cd90c3882dd27aa99f12137abfe35bc0027e067d

File: client/src/components/EpisodeCard.js (hunk 4 above)
Comment (ncbecker) [reply client/src/components/EpisodeCard.js]: Created an example story with adjustable props in storybook https://github.com/ncbecker/podium/pull/42/commits/b0040da4325f8932f32266830556dfc3b928033c

File: client/src/components/EpisodeSearch.js [hunk 5]
```diff
@@ -3,9 +3,10 @@ import styled from "styled-components/macro";
 import PropTypes from "prop-types";
 
 const InputField = styled.input`
-  width: 280px;
+  width: 265px;
```
Comment (phlgr) [inline client/src/components/EpisodeSearch.js:6]: Why do you use px here instead of % and use px for larger viewports.
Try researching media queries

Comment (ncbecker) [reply client/src/components/EpisodeSearch.js:6]: Thanks for your hint! I set up a new issue to make the whole content responsive for bigger screens at a later stage 👍

**Pull request merged**